projects/utils-ogc/src/lib/wps/wps100/wps_marshaller_1.0.0.ts

```diff
@@ -135,7 +135,9 @@ export class WpsMarshaller100 implements WpsMarshaller {
     if (data.complexData) {
 
       if (data.complexData.encoding === 'base64') {
+        // @ts-ignore
         if (typeof module !== 'undefined' && module.exports) { // node
+          // @ts-ignore
```

Comment:
If this ignore is because of `new Buffer`  you can use [`Buffer.from()`](https://nodejs.org/en/docs/guides/buffer-constructor-deprecation/) it then gets converted by something like https://github.com/feross/buffer in the build process.